install/linux/usr/share/odemis/sim/sparc2-fslm-sim.odm.yaml

```diff
@@ -0,0 +1,340 @@
+# Configuration for a simulated SPARC v2 system with a free-space-laser module (FSLM)
+SPARC2-FSLM: {
+    class: Microscope,
+    role: sparc2,
+    children: ["SEM E-beam", "SEM Detector", "Calibration Light",
+               "Optical Path Properties", "Integrated Spectrometer", "External Spectrometer",
+               "Camera", "External Camera",
+               "Spec Filter Wheel", "Spectrometer Selector", "Spectrograph focus",
+               "Mirror Actuators",
+               "Slit",
+               "Lens1 Mover", "Lens2 Switch",
+               "In-Light Aligner",
+              ],
+}
+
+# Light (lamp with known spectrum)
+"Calibration Light": {
+    class: light.Light,
+    role: "brightlight",
+    power_supplier: "Power Control Unit",
+    affects: ["Camera", "Integrated Spectrometer"],
```

Comment:
I don't know where exactly it's situated on this module, but if it's before the "spec-switch", it can also affect the External Camera/Spectrometer.